dissect/target/plugins/os/windows/datetime.py

```diff
@@ -194,14 +198,26 @@ class DateTimePlugin(Plugin):
     def __init__(self, target):
         super().__init__(target)
 
-        tz_info_key = "HKLM\\SYSTEM\\CurrentControlSet\\Control\\TimeZoneInformation"
+        tz_name = None
 
+        timezone_information = self.target.registry.key("HKLM\\SYSTEM\\CurrentControlSet\\Control\\TimeZoneInformation")
         try:
-            tz_name = self.target.registry.key(tz_info_key).value("TimeZoneKeyName").value
-            self._tzinfo = self.tz(tz_name)
-        except RegistryError:
+            tz_name = timezone_information.value("TimeZoneKeyName").value  # Windows 7+
+        except RegistryValueNotFoundError:
+            tz_name_localized = timezone_information.value(
+                "StandardName"
+            ).value  # < Windows 7 (https://github.com/dateutil/dateutil/issues/210)
```

Comment:
```suggestion
            # Windows 7+
            tz_name = timezone_information.value("TimeZoneKeyName").value
        except RegistryValueNotFoundError:
            # < Windows 7 (https://github.com/dateutil/dateutil/issues/210)
            tz_name_localized = timezone_information.value("StandardName").value
```
I assume this makes black happy and makes the line a bit easier to read.